### ISSUE TITLE
Save only open files

### ISSUE THREAD

**Issue by Slender1808** [syncthing/syncthing#6753]
It is possible to synchronize a folder leaving only the files I open saved only

Like Resilio does it?

**Comment by calmh** [comment]
I don’t understand this at all. Can you rephrase this somehow?

**Comment by Slender1808** [comment]
I do not know the technical term of this but it would be something like that, the synchronization would save a kind of shortcut instead of saving the file itself, and only when the file was opened that the synchronization would download the complete file

**Comment by AudriusButkevicius** [comment]
I guess you are talking about Windows placeholder files that are used by OneDrive etc.

I don't think we will ever support this, it would have to be a separate windows specific application.

**Comment by calmh** [comment]
Yeah, no. On demand download isn't on the radar for Syncthing.

**Comment by Slender1808** [comment]
I was talking about it Selective Sync Mode

https://help.resilio.com/hc/en-us/articles/205457775-Synchronization-Modes

**Comment by AudriusButkevicius** [comment]
There already is a ticket for that.

**Comment by Slender1808** [comment]
how great I didn't know

**Comment by Qifcn** [comment]
The on-demand synchronization function is a great function, which will open up more application scenarios of syncthing, and this function is not difficult to implement. I am sorry that there is no code capability, but it is viewed from the perspective of a user.  Although there is a file ignoring function, the above is the inappropriateness of Google Translate.
